encode.js

```diff
@@ -5,34 +5,52 @@ const mri = require('mri')
 const pkg = require('./package.json')
 
 const argv = mri(process.argv.slice(2), {
+    alias: {
+        help: ['h'],
+        version: ['v'],
+    },
 	boolean: [
-		'help', 'h',
-		'version', 'v',
+		'help',
+		'version',
 	]
 })
 
-if (argv.help || argv.h) {
-	process.stdout.write(`
-Usage:
+const usage = `Usage:
     echo -n <url> | url-encode
+    or: url-encode [FILE]
```

Comment:
From my experience, mandatory positional arguments are often specified as `<file>` and optional positional arguments as `[file]`. Shall we follow this convention?

```suggestion
    or: url-encode <FILE>
```

Reply:
It's not mandatory, though.

Reply:
If you consider manually providing input via `stdin` a use case, yes.